### PR TITLE
Add proposal of mapper supporting stream data 

### PR DESCRIPTION
**Pull request by wbc6080**
**What type of PR is this?**
/kind design

**What this PR does / why we need it**:
Add proposal of mapper supporting video stream data
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by wbc6080** [comment]
@Shelley-BaoYue @zhiyingfang2022 Please help add this pr to version 1.17, thanks

**Comment by zhiyingfang2022** [comment]
> @Shelley-BaoYue @zhiyingfang2022 Please help add this pr to version 1.17, thanks

ok,done.

File: docs/proposals/mapper-support-streaming-data.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+title: Mapper Support Streaming Data
```
Comment (fisherxu) [inline docs/proposals/mapper-support-streaming-data.md:2]: The proposal is mainly focus on the `video stream data`, and stream data should include many other types, so forr the title, how about change to `Mapper Support Video Streaming Data`?  cc @WillardHu @wbc6080 HDYT

Comment (wbc6080) [reply docs/proposals/mapper-support-streaming-data.md]: Done @fisherxu

File: docs/proposals/mapper-support-video-streaming-data.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+- Mapper is able to obtain video streams from camera devices in edge nodes.
+- Add streaming data processing capabilities to the mapper-framework data plane.
+- Mapper is able to forward video streams from edge to the cloud. (still in planning, may support in future)
+
+
+## Design Details
+
+### Architecture
+
+<img src="../images/mapper/onvif-mapper.png">
```
Comment (Shelley-BaoYue) [inline docs/proposals/mapper-support-video-streaming-data.md:31]: I cannot load the image

Comment (wbc6080) [reply docs/proposals/mapper-support-video-streaming-data.md]: Sorry, I submitted the old version of the document by mistake. Now the document has been corrected. @Shelley-BaoYue

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**